keps/sig-storage/1710-selinux-relabeling/README.md

```diff
@@ -446,19 +441,26 @@ _This section must be completed when targeting beta graduation to a release._
   Try to be as paranoid as possible - e.g. what if some components will restart
   in the middle of rollout?
   
-  Running workloads are not affected during rollout, because they don't use the new API fields. 
+  This KEP affects only kubelet behavior and only mounts on the node where kubelet runs.
+  Different nodes in a cluster can have the feature enabled/disabled without any issues.
 
 * **What specific metrics should inform a rollback?**
 
+  TBD: We propose a metric above, file its name here.
```

Comment:
heh. forgot to fill this in?

Reply:
This should not be required for Alpha. We'll figure it out later.